src/debug_menu.cpp

```diff
@@ -121,99 +128,145 @@ class mission_debug
         static std::string describe( const mission &m );
 };
 
-static int player_uilist()
+/**
+ * Filter entries to be displayed in a menu.
+ * @param entries: A vector where each element is a pair consisting of (first) a menu entry and (second) boolean indicating whether to show the entry or not.
+ * @param display_all_entries: boolean used to override the entries in the @p entries vector. If `true` then all entries are shown (no matter what is their boolean state);
+ *  if `false` then the boolean of the menu entry is taken into account.
+ * @returns A filtered vector of menu entries, depending on the given parameters.
+ */
+std::vector<uilist_entry> filter_uilist( const std::vector<uilist_entry_pair_t> &entries,
```

Comment:
```suggestion
static std::vector<uilist_entry> filter_uilist( const std::vector<uilist_entry_pair_t> &entries,
```
(May need to change where line breaks too)